src/config/have_sys_socket.c

```diff
@@ -243,7 +243,7 @@ sys_socket_sendto_target( struct network_target *target,
   result = send( target->handle,
                  msg,
                  msg_length,
-                 0 );
+                 MSG_NOSIGNAL );
```

Comment:
A quick note on this: it wasn't clear to me how this flag would behave on Windows (it's not listed as a flag in winsock2.h at least) and I didn't try it out while I was doing research on the issue.

Reply:
Ah yes, thanks for pointing that out! I did run the test suites on a Windows machine before adding the fix, and winsock2 seems to return an error with send immediately (and doesn't end the program). This is different from POSIX systems with `sys/socket.h` where a send loop was needed to eventually capture the error (or cause termination before the flag was added). That is also why I added the roadmap to do some more error checking to get a faster "time to error" out of the POSIX side of things.

All that to say that the `src/config/have_winsock2.c` code that handles network target sending for Windows/Winsock builds is unchanged here for that reason. The test suite does still run on Windows builds just to make sure.